File: output/CodeGen/avatar/Java/Python/codeforces_190_A.py
def main(args):
    sc = Scanner(System.in)
    n = sc.nextInt()
    m = sc.nextInt()
    if n == 0:
        if m!= 0:
            print("Impossible")
        else:
            print(0 + " " + 0)
        return
    if m <= n:
        if m == 0:
            m = 1
        print(n + " " + (m + n - 1))
        return
    print(m + " " + (m + n - 1))

File: output/CodeGen/avatar/Python/Java_source/codeforces_190_A.java
public class codeforces_190_A {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int m = in.nextInt();
        if (n < 1) {
            if (m < 1) {
                System.out.println(n + " " + m);
            } else {
                System.out.println("Impossible");
            }
        } else if (m > n) {
            int min = m;
            int max = n + m - 1;
            System.out.println(min + " " + max);
        } else {
            int min = n;
            int max = n + m - 1;
            if (m == 0) {
                max = n;
            }
            System.out.println(min + " " + max);
        }
    }
}